tests/various/hierarchy_defer.ys

```diff
@@ -0,0 +1,19 @@
+read -vlog2k <<EOT
```

Comment:
This should either be preceded by `read -noverific` or it should be `read_verilog` instead of `read`. Because here document don't work with Verific. (Yet. Feel free to add support for that.)

Reply:
The difference between `read` and `read_verilog` is that the former calls `read_verilog -defer`. Let me add `read -noverific` then.